src/main/java/de/tum/in/www1/artemis/web/rest/publicc/AtheneResource.java

```diff
@@ -32,14 +32,16 @@ public AtheneResource(AtheneService atheneService) {
     }
 
     /**
-     * Saves automatic textAssessments of Athene
+     * POST athene-result/:exerciseId -- Saves automatic textAssessments of Athene
      *
      * @param exerciseId The exerciseId of the exercise which will be saved
      * @param atheneResponse The calculation results containing blocks and clusters
      * @param auth The secret for authorization
      * @return 200 Ok if successful or 401 unauthorized if secret is wrong
      */
-    @PostMapping(value = "/{exerciseId}", consumes = "application/x-protobuf")
+    // TODO: /public
+    @PostMapping(value = "athene-result/{exerciseId}", consumes = "application/x-protobuf")
```

Comment:
Would this URL have to updated in Athena code somewhere (⇒ open issue there)?

Should there be a non-`/public/` endpoint in the `LegacyResource` for now that just redirects here to one that includes `/public/`? Then the transition might be easier.

Reply:
I think it's okay because we will specify in the release notes. Institutions using Athene can then simple update the config and everything rolls.
We have the LegacyResource because it's more complicated for the whole repositories.

@krusche Do you agree with me or would you rather have such a fallback endpoint for Athene too?

Reply:
@jpbernius @maximiliansoelch what do you think? can we change Athene quickly after merging this PR?

@julian-christl the path contains a typo "publicc"

Reply:
@krusche 
As written in the PR description:
> The package name for public resources is `publicc`. The reason is that `public` is protected and can't be used. I did not see a perfect solution for that so I settled with this option. If you have a better suggestion on how to solve the problem, please let me know (e.g. a better term).

Reply:
We do not need to change `Athena` at all. Artemis specifies the Callback URL. Make sure to update the usage as well: https://github.com/ls1intum/Artemis/blob/develop/src/main/java/de/tum/in/www1/artemis/service/connectors/athene/AtheneService.java#L165

Reply:
From what I see in the code, that is already the case. I'll look into this again in detail during daytime.

Reply:
@jpbernius That's correct. I changed it here: https://github.com/ls1intum/Artemis/pull/6165/files#diff-c991a8c31e27a496b18e6f1541d635f623481d598658bbaf9c190898ef09ec85

Thank you, good to know that there is no additional change required.